src/__tests__/linter.jest.test.ts

```diff
@@ -170,4 +174,176 @@ describe('Linter', () => {
       );
     });
   });
+
+  describe('Exceptions handling', () => {
+    it('should ignore specified rules violations in a standalone document', async () => {
+      await spectral.loadRuleset(customOASRuleset);
+      spectral.registerFormat('oas3', isOpenApiv3);
+
+      const res = await spectral.run(
+        {
+          openapi: '3.0.2',
+          info: 17,
+        },
+        {
+          resolve: {
+            documentUri: '/test/file.json',
+          },
+        },
+      );
+
+      expect(res.length).toBeGreaterThan(0);
+
+      expect(res).not.toContainEqual(
+        expect.objectContaining({
+          code: 'info-contact',
+        }),
+      );
+
+      expect(res).not.toContainEqual(
+        expect.objectContaining({
+          code: 'info-description',
+        }),
+      );
+
+      expect(res).not.toContainEqual(
+        expect.objectContaining({
+          code: 'oas3-api-servers',
+        }),
+      );
+    });
+
+    describe('resolving', () => {
+      const testRules: Dictionary<Rule, string> = {
```

Comment:
@P0lip I now realize this whole pre-test setup is a lot of code. Do you want me to try and repackage it, or is it good enough?

Reply:
Yeah, I believe we could create a simple ruleset a put in a different file and simply load it here via `loadRuleset`? What do you think?

Reply:
@P0lip Fixed